src/radical/pilot/utils/component.py

```diff
@@ -378,8 +378,9 @@ def _control_cb(self, topic, msg):
             with self._cancel_lock:
                 self._cancel_list += uids
 
-            # scheduler handles cancelation itself
-            if 'AgentSchedulingComponent' in repr(self):
+            # scheduler and executor handle cancelation directly
+            if 'agent.scheduling' in repr(self) or \
```

Comment:
`agent.scheduling` -> `agent.scheduler`

Reply:
Funny, I never realized that this directory is inconsistently named...  Thanks for noting it :-)